debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -712,53 +654,36 @@ else if (data instanceof PGobject) {
      * @return a value which will be used by Connect to represent the actual point value
      */
     protected Object convertPoint(Column column, Field fieldDefn, Object data) {
-        if (data == null) {
-            data = fieldDefn.schema().defaultValue();
-        }
-        Schema schema = fieldDefn.schema();
-        if (data == null) {
-            if (column.isOptional()) return null;
-            return handleUnknownData(column, fieldDefn, data);
-        }
-        if (data instanceof PGpoint) {
-            PGpoint pgPoint = (PGpoint) data;
-            return Point.createValue(schema, pgPoint.x, pgPoint.y);
-        }
-        if (data instanceof String) {
-            String dataString = data.toString();
-            try {
-                PGpoint pgPoint = new PGpoint(dataString);
-                return Point.createValue(schema, pgPoint.x, pgPoint.y);
-            } catch (SQLException e) {
-                logger.warn("Error converting the string '{}' to a PGPoint type for the column '{}'", dataString, column);
+        return convertValue(column, fieldDefn, (Supplier<?>)() -> handleUnknownData(column, fieldDefn, data), Collections.emptyList(), (r) -> {
```

Comment:
Hum, we know the value is null at this point, right? What's the benefit for passing it through `handleUnknownData()`? Why not just pass the (0, 0) point in this case?

Reply:
Looks like a copy/paste error, fixing.